test/scrabble_test.rb

```diff
@@ -6,8 +6,23 @@
 
 class ScrabbleTest < Minitest::Test
   def test_it_can_score_a_single_letter
-    skip
     assert_equal 1, Scrabble.new.score("a")
     assert_equal 4, Scrabble.new.score("f")
   end
+
+  def test_it_can_score_two_letters
+    assert_equal 4, Scrabble.new.score("be")
+    assert_equal 2, Scrabble.new.score("no")
+  end
+
+  def test_it_can_score_a_full_word
+    assert_equal 8, Scrabble.new.score("hello")
+    assert_equal 16, Scrabble.new.score("zebra")
+  end
+
+  def test_it_can_score_nil_or_empty_string_as_zero
+    assert_equal 0, Scrabble.new.score("")
+    assert_equal 0, Scrabble.new.score(nil)
+  end
```

Comment:
Like that these tests seem to come directly from the requirements. I can't tell for sure, but it seems that you actually used these to drive development, which would be great.

Splitting the `#score` method into smaller supporting methods would allow you to have some unit tests.